qoffer-domain/src/main/groovy/life/qbic/portal/portlet/SearchCriteria.groovy

```diff
@@ -11,11 +11,9 @@ package life.qbic.portal.portlet
  */
 class SearchCriteria {
 
-    final CriteriaType criteriaType
-    final String criteriaValue
+    final HashMap<CriteriaType,String> criteria
```

Comment:
```suggestion
    final Map<CriteriaType,String> criteria
```
Nice! now we can have multiple criteria. You can make the declaration more generic here.